SteamBundleSitesExtension.js

```diff
@@ -842,12 +878,40 @@ const siteHandlers = {
         GM_addStyle(`
             .SBSE_container { margin-top: 10px;}
             .SBSE_container > textarea { border: 1px solid #CC001D;}
-            .SBSE_container button { background-color: #CC001D; color: white;}
+            .SBSE_container button { background-color: #CC001D; color: white; border-radius: 3px;}
+            .SBSE_BtnExport { background-color: #CC001D; color: white; border-radius: 3px;}
+            .SBSE_BtnExport:hover {color: white;}
```

Comment:
I think this is redundant. '.SBSE_BtnExport' already has style of color white

Reply:
not white when hover

Reply:
Ah sorry, Chrome developer tool didn't show it when hover, thought it's not applying

Reply:
It inherited color when hover is black
If do not set this style, text color will turn to black when you hover on it.